firmware/arduino/chessboard/movementFunctions.ino

```diff
@@ -13,3 +13,20 @@ bool alignPiece(int col, int row)
     return true;
 }
 
+// Assumes that the starting position is the center of the square the piece is on
+void centerPiece()
+{
+    moveToFirstCircle();
+
+    // Enable electromagnet
+    analogWrite(ELECTROMAGNET, PWM_SCALE);
+
+    for (int i = 0; i < NUM_CIRCLES; i++)
+    {
+        makeCircle(i);
+        moveToNextCircle(i);
+    }
+
+    // Turn electromagnet off
+    analogWrite(ELECTROMAGNET, 0);
```

Comment:
First off, it's good that you used 0 instead of LOW here (even though they're the same thing), cuz 0 implies a frequency.
It would still be a good idea to make a preprocessor for 0 if you used analogWrite.

However, based on my experience, digitalWriting LOW is better than analogWriting 0. Plus you can use a built-in preprocessor from arduino :)